src/main/steps/upload-documents/template.njk

```diff
@@ -44,19 +44,22 @@
 
 <p class="govuk-body">{{paragraph2}}</p>
 <p class="govuk-body">{{paragraph3}}</p>
-  <ul class="govuk-list govuk-list--bullet">
+<p class="govuk-body">{{uploadedDocumentsSubTitle}}</p>
+<div class="govuk-body">
+  <ul class="‘govuk-list govuk-list--bullet’">
     {% for item in uploadDocumentsBullets %}
       <li>{{item | safe}}</li>
     {% endfor %}
   </ul>
+</div>
 <p class="govuk-body">{{caseRelevancySubTitle}}</p>
 
 
 {% set fileRequirements = ""%}
 {% for item in uploadHelpBullets %}
     {% set fileRequirements = fileRequirements + "<li>" + item + "</li>"%}
 {% endfor %}
-{% set uploadRequirements = "<ul class='govuk-list govuk-list--bullet'><div class='govuk-body-s'>" + fileRequirements + "</div></ul><p class='govuk-body-s'>"+ guideText + "</p>" %}
+{% set uploadRequirements = "<ul class='‘govuk-list govuk-list--bullet’'><div class='govuk-body-s'>" + fileRequirements + "</div></ul><p class='govuk-body-s'>"+ guideText + "</p>" %}
```

Comment:
how comes there are two speech marks instead of one surrounding govuk-list govuk-list--bullet?

Reply:
To match sptribs-frontend. When rendered it causes the bullet points to be indented.